src/gui/MixerView.cpp

```diff
@@ -590,7 +590,7 @@ void MixerView::setCurrentMixerChannel(int channel)
 
 void MixerView::clear()
 {
-	getMixer()->clear();
+	while (m_mixerChannelViews.size() > 1) { deleteChannel(1); }
```

Comment:
This won't clear the master channel while `Mixer::clear` does. Also, why don't you delete channels from back?

Reply:
Ah, good catch. I need to call `clearChannel(0)` still. I'll get that added.

> Also, why don't you delete channels from back?

Because `deleteChannel` handles everything fine, but it may be better on the vector processing to go back to front.